erc20/ds-token/ds-token-erc20-spec.ini

```diff
@@ -1,265 +1,301 @@
+gasCap: 100000

```

Comment:
why the diff shows entire file is changed? did you change the file format  dos <-> unix  stuff?

Reply:
Looks it was indeed CRLF instead of LF. I fixed it.